### PR TITLE
Adding info on Dart obfuscation

### PR DESCRIPTION
**Pull request by Sfshaza**
Addresses https://github.com/flutter/flutter/issues/18137
(FYI, @Ehud-Banunu)

Staged:
(Android) https://sz-flutter.firebaseapp.com/android-release/#obfuscate-your-dart-code
(iOS)  https://sz-flutter.firebaseapp.com/ios-release/#obfuscate-your-dart-code

### CODE REVIEW

File: android-release-build.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ file.
 Release builds of your app will now automatically be signed.
 
 
-## Minify and obfuscate
+## Minify and obfuscate the Android wrapper
```
Comment (mit-mit) [inline android-release-build.md]: I believe our usual term for this is 'Android host'

File: ios-release-build.md [hunk 2]
```diff
@@ -11,7 +11,53 @@ the [App Store][appstore] and [TestFlight][testflight].
...
+if [[ -n "$EXTRA_GEN_SNAPSHOT_OPTIONS" ]]; then
+  extra_gen_snapshot_options_or_none="--extra-gen-snapshot-options=$EXTRA_GEN_SNAPSHOT_OPTIONS"
+fi
+```
+
+### Step 2 - Apply your changes
+
+In the project root of your app, run the following two commands:
+
+```
```
Comment (Ehud-Banunu) [inline ios-release-build.md]: Please remove the "run" prefix

File: ios-release-build.md [hunk 3]
```diff
@@ -11,7 +11,53 @@ the [App Store][appstore] and [TestFlight][testflight].
...
+run git commit -am "Enable obfuscation on iOS"
+flutter
+```
+
+### Step 3 - Modify the release config
+
```
Comment (Ehud-Banunu) [inline ios-release-build.md]: Please remove the trailing ">" after "Release.xcconfig"

**Comment by xster** [comment]
IMO we should just call this section 'Proguard' instead of paraphrasing it. Android developers would understand what it's referring to there's no risk of causing reader to misinterpret it as targeting anything other than JVM bytecode.

**Comment by Sfshaza** [comment]
As Chris requested, I've moved the details on obfuscating Dart code to the [Flutter wiki](https://github.com/flutter/flutter/wiki/Obfuscating-Dart-Code).

**Comment by Sfshaza** [comment]
Accidentally did some work on master, which caused me to create PR https://github.com/flutter/website/pull/1081 for this issue.  Closing this.

**Pull request closed without merging**